.github/workflows/comment-on-asciidoc-changes.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: Comment on PR for .asciidoc changes
+
+on:
+  pull_request:
+    types:
+      - synchronize
```

Comment:
This may end up leaving quite a few duplicate comments.  I'm assuming it's to catch already open pull requests, but wanted to make a note.